content/applications/hr/employees/certifications.rst

```diff
@@ -0,0 +1,117 @@
+   report. All other certifications appear in the resume section of the :doc:`employee form
+   <new_employee>`.
+
+View certifications by expiration status
+----------------------------------------
+
+When managing a large number of employees with a variety of certifications, it can be difficult to
+determine which employees need to keep necessary certifications current in the default list view. In
+this scenario, it is beneficial to view the certifications by expiration status.
+
+To do so, navigate to :menuselection:`Employees app --> Reporting --> Certifications`. Next, click
+the :icon:`fa-caret-down` :guilabel:`(down arrow)` in the search bar, then click :guilabel:`Add
+Custom Group`, revealing a drop-down menu. Click :guilabel:`Expiration Status`, then click away from
+the drop-down menu to close it.
+
+After doing so, all the certifications are organized by status, starting with :guilabel:`Expired`
+certifications, then certifications that are :guilabel:`Expiring` soon (within the next 90 days),
+and lastly, certifications that are still :guilabel:`Valid`.
+
+This view provides an easy way to see which employees have certifications that are going to expire
+soon, to determine which employees need to take action to keep their certifications current.
+
+.. image:: certifications/status.png
+   :align: center
+   :alt: The list of employee certifications, grouped by status.
+
+.. _certifications/form:
+
+Log a certification
+===================
+
+To log a certification for an employee, navigate to :menuselection:`Employees app --> Reporting -->
+Certifications`. Click :guilabel:`New`, and a blank certification form loads. Enter the following
+information on the form:
+
+- :guilabel:`Title`: Enter a short description for the certification in this field.
+- :guilabel:`Employee`: Using the drop-down menu, select the employee who received the
+  certification.
+- :guilabel:`Type`: Using the drop-down menu, select the type of certification received. This field
+  determines where on the employee's resume the certification appears. To create a new
+  :guilabel:`Type`, enter the type in the field, then click :guilabel:`Create "type"`.
+
+  The default options are:
+
+   - :guilabel:`Experience`: Select this option to have the certification appear in the *Experience*
+     section of the *Resume* tab on the :doc:`employee form <new_employee>`.
+   - :guilabel:`Education`: Select this option to have the certification appear in the *Education*
+     section of the *Resume* tab on the :doc:`employee form <new_employee>`.
+   - :guilabel:`Internal Certification`: Select this option to have the certification appear in the
+     *nternal Certification* section of the *Resume* tab on the :doc:`employee form <new_employee>`.
+   - :guilabel:`Completed Internal Training`: Select this option to have the certification appear in
+     *Completed Internal Training* section of the *Resume* tab on the :doc:`employee form
+     <new_employee>`.
```

Comment:
Scoot back a space to align with the parent bullet point, also, fix "nternal" typo :)
```suggestion
  - :guilabel:`Experience`: Select this option to have the certification appear in the *Experience*
    section of the *Resume* tab on the :doc:`employee form <new_employee>`.
  - :guilabel:`Education`: Select this option to have the certification appear in the *Education*
    section of the *Resume* tab on the :doc:`employee form <new_employee>`.
  - :guilabel:`Internal Certification`: Select this option to have the certification appear in the
    *Internal Certification* section of the *Resume* tab on the :doc:`employee form <new_employee>`.
  - :guilabel:`Completed Internal Training`: Select this option to have the certification appear in
    *Completed Internal Training* section of the *Resume* tab on the :doc:`employee form
    <new_employee>`.
```